src/Tasks/Microsoft.Common.props

```diff
@@ -63,72 +63,19 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   <Import Project="$(MSBuildProjectExtensionsPath)$(MSBuildProjectFile).*.props" Condition="'$(ImportProjectExtensionProps)' == 'true' and exists('$(MSBuildProjectExtensionsPath)')" />
 
   <!--
-        Import wildcard "ImportBefore" props files if we're actually in a 12.0+ project (rather than a project being
-        treated as 4.0)
+          Wildcard imports come from $(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props\ folder.
```

Comment:
Removed the reference to the "Microsoft.Common.props.d folder" since as far as I can tell, it never actually existed? Even in the PR which added this comment, it didn't seem to apply then either.

Reply:
The reason for the weird naming is given the comments itself.

> Unfortunately, there is already a file named "Microsoft.Common.props" in this directory so we have to have a slightly different directory name to hold extensions.